### PR TITLE
(style) Landing Page and components styled. Functions Correctly

### PR DESCRIPTION
**Pull request by lschwall**

### CODE REVIEW

**Review by tenhold — APPROVED** [review]

File: server/routes/owner.js [hunk 1]
```diff
@@ -132,14 +132,14 @@ ownerRouter.post('/id', (req, res) => {
   Owner.findAll({ where: { email } })
     .then((owner) => {
       if (owner.length > 0) {
-        res.send(JSON.stringify(owner[0].id))
+        res.send(JSON.stringify(owner[0].id));
       } else {
-        res.sendStatus(500)
-        console.info('OWNER NOT FOUND')
+        res.sendStatus(500);
+        console.info('OWNER NOT FOUND');
```
Comment (jstrzesz) [inline server/routes/owner.js]: remove

File: src/components/SignIn/owner/OwnerCredentials.jsx [hunk 2]
```diff
@@ -14,48 +26,66 @@ const useStyles = makeStyles(() => ({
...
+          method: 'POST',
+          url: '/db/owner/is-verify',
+          headers: {
+            token: localStorage.ownerToken,
+          },
+        })
+          .then(({ data }) => {
+            if (data) history.push('/owner');
+          })
+          .catch((err) => console.warn(err));
```
Comment (jstrzesz) [inline src/components/SignIn/owner/OwnerCredentials.jsx]: dont need nested .catch blocks

File: src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx [hunk 3]
```diff
@@ -91,7 +111,7 @@ const BarSearch = ({
   const showSelection = () => {
     if (selected) {
       return (
-        <Grid item container direction="row" justify="center" alignItems="center">
+        <Grid item container direction="row" justify="center" alignItems="center" className={classes.selection}>
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx]: put each attribute on its own line

File: src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx [hunk 4]
```diff
@@ -126,9 +146,22 @@ const BarSearch = ({
   return (
     <Grid container direction="column" justify="center" alignItems="center">
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx]: attributes on separates lines

File: src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx [hunk 5]
```diff
@@ -140,15 +173,40 @@ const BarSearch = ({
       </Grid>
       <Grid item container direction="row" justify="center" alignItems="center">
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx]: see above comments

File: src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx [hunk 6]
```diff
@@ -140,15 +173,40 @@ const BarSearch = ({
...
+        justify="center"
+        alignItems="center"
+        className={classes.buttons}
+      >
+        <Button
+          size="small"
+          color="primary"
+          className={classes.backBtn}
+          onClick={() => {
+            setLandingView('signUp');
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/BarSearch.jsx:203]: tab back

File: src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx [hunk 7]
```diff
@@ -102,26 +118,52 @@ const OwnerInfo = ({
 
   return (
     <Grid container direction="column" justify="center" column="center">
-      <Grid item container direction="row" justify="center" column="center">
-        <Typography variant="subtitle1">
-          Owner Information
-        </Typography>
+      <Grid item container direction="row" justify="center" column="center" className={classes.textField}>
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx]: see above comments

File: src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx [hunk 8]
```diff
@@ -102,26 +118,52 @@ const OwnerInfo = ({
...
+          id="outlined-basic"
+          variant="outlined"
+          required
+          label="User Name"
+          onChange={(e) => { setUsername(e.target.value); }}
+        />
       </Grid>
-      <Grid item container direction="row" justify="center" column="center">
-        <TextField id="standard-basic" label="User Name" onChange={(e) => { setUsername(e.target.value); }} />
+      <Grid item container direction="row" justify="center" column="center" className={classes.textField}>
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx]: see above comments

File: src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx [hunk 9]
```diff
@@ -102,26 +118,52 @@ const OwnerInfo = ({
...
+          variant="outlined"
+          required
+          label="First Name"
+          onChange={(e) => { setFirst(e.target.value); }}
+        />
       </Grid>
-      <Grid item container direction="row" justify="center" column="center">
-        <TextField id="standard-basic" label="First Name" onChange={(e) => { setFirst(e.target.value); }} />
-        <TextField id="standard-basic" label="Last Name" onChange={(e) => { setLast(e.target.value); }} />
+      <Grid item container direction="row" justify="center" column="center" className={classes.textField}>
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx]: see above comments

File: src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx [hunk 10]
```diff
@@ -102,26 +118,52 @@ const OwnerInfo = ({
...
+          id="outlined-basic"
+          variant="outlined"
+          required
+          label="Last Name"
+          onChange={(e) => { setLast(e.target.value); }}
+        />
       </Grid>
-      <Grid item container direction="row" justify="center" column="center">
-        <TextField id="standard-basic" label="Email" onChange={(e) => { setEmail(e.target.value); }} />
+      <Grid item container direction="row" justify="center" column="center" className={classes.textField}>
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx]: see above comments

File: src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx [hunk 11]
```diff
@@ -102,26 +118,52 @@ const OwnerInfo = ({
...
+          label="Email"
+          onChange={(e) => { setEmail(e.target.value); }}
+        />
       </Grid>
-      <Grid item container direction="row" justify="center" alignItems="center">
-        <FormControl className={clsx(classes.margin, classes.textField)}>
-          <InputLabel htmlFor="standard-adornment-password"> Choose Password</InputLabel>
-          <Input
-            id="standard-adornment-password"
+      <Grid item container direction="row" justify="center" alignItems="center" column="center">
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx]: see above comments

File: src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx [hunk 12]
```diff
@@ -131,22 +173,38 @@ const OwnerInfo = ({
...
         }}
       >
         Submit
       </Button>
+      <Button
+        size="small"
+        color="primary"
+        className={classes.backBtn}
+        onClick={() => {
+          handleBack();
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/OwnerInfo.jsx:202]: tab back

File: src/components/SignUp/Form/Owner/Form Components/SearchBox.jsx [hunk 13]
```diff
@@ -1,95 +1,115 @@
...
+      <Combobox
+        onSelect={async (address) => {
+          setValue(address, false);
+          clearSuggestions();
+          try {
+            const results = await getGeocode({ address });
+            const { place_id } = results[0];
+            const details = await getDetails({ placeId: place_id });
+            getPlaceInfo(details);
+            // console.info('this is results', results);
```
Comment (jstrzesz) [inline src/components/SignUp/Form/Owner/Form Components/SearchBox.jsx]: remove

File: src/components/SignUp/Form/customer/CustomerSignUpForm.jsx [hunk 14]
```diff
@@ -68,130 +101,66 @@ const CustomerSignUpForm = ({
...
     }
     return (
-      <SafetyDialog setCounter={setCounter} />
+      <SafetyDialog setCounter={setCounter} setLandingView={setLandingView} />
     );
   };
 
   return (
     <Grid container direction="column" justify="center" alignItems="center">
+      <Grid item container direction="row" justify="center" alignItems="center">
```
Comment (jstrzesz) [inline src/components/SignUp/Form/customer/CustomerSignUpForm.jsx:150]: see above comments

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

**Review by christophermbooth — APPROVED** [review]

**Review by Bri-B — APPROVED** [review]

**Pull request merged**